### PR TITLE
Add a promo banner for 29th March 2017

### PR DESCRIPTION
**Pull request by boffbowsh**

### CODE REVIEW

**Comment by robinwhittleton** [comment]
Shame we’re using a `'` instead of a `’` but hey ho.

**Comment by dracos** [comment]
@robinwhittleton PR at https://github.com/alphagov/frontend/pull/1202 ;)

File: app/views/homepage/_promo_banner.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div id="homepage-promo-banner">
+  <div class="banner-message">
+    <p>The government has published its plan for the UK leaving the EU.<br /><a href="https://www.planforbritain.gov.uk/news/prime-ministers-letter-donald-tusk-triggering-article-50/" rel="external nofollow">Read the Prime&nbsp;Minister's letter</a>.</p>
```
Comment (nickcolley) [inline app/views/homepage/_promo_banner.html.erb]: fyi https://github.com/alphagov/frontend/issues/1095

Comment (boffbowsh) [inline app/views/homepage/_promo_banner.html.erb]: i'm not sure what you mean. This link doesn't open in a new window.

Comment (nickcolley) [inline app/views/homepage/_promo_banner.html.erb]: teach me to comment when I should be sleeping haha

**Pull request merged**